examples/nvingest_mcp/pyproject.toml

```diff
@@ -0,0 +1,41 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024-2025, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools >= 64"]
+
+[project]
+name = "nvingest_mcp"
+version = "0.1.0"
+dependencies = [
+    "nvidia-nat[mcp,langchain]",
```

Comment:
Pls pin the version to ensure the example continues to work consistently. Use two digits for NAT `nvidia-nat[mcp,langchain]~=1.3`
Other packages may also need to be pinned (as there is no uv.lock here)